### DISCUSSION TITLE
Invert colors to show active part of window

### DISCUSSION THREAD

**Discussion by tmchow** [rxhanson/RectanglePro-Community#294] [Ideas]
In the current design, the "active" part of the thumbnail showing where the window will snap to is opposite of what I think it should be.  Currently it uses the darker region to show the active part, when I think the lighter part should be.  Example:

<img width="92" alt="image" src="https://github.com/rxhanson/Rectangle/assets/517103/11d8aa9b-b464-4751-9bbb-4f49c84ef388">

Every time I see this UI or the in the menu bar preferences I'm confused and double take on which is the right option.

**Comment by rxhanson** [comment]
Thanks for bringing this up. It’s on my todo list, just haven’t gotten to it yet (there are a lot of icons to change)

**Reply by tmchow** [reply]
Any update on this?

**Reply by rxhanson** [reply]
Thanks for checking in! No updates at the moment, sorry.

**Reply by tmchow** [reply]
Checking back in @rxhanson :)
